src/org/traccar/model/CellTower.java

```diff
@@ -106,4 +106,10 @@ public void setSignalStrength(Integer signalStrength) {
         this.signalStrength = signalStrength;
     }
 
+    public void setOperator(long operator) {
```

Comment:
Is it a common way of representing the data? If it's specific to Teltonika, it should be in the decoder.

Reply:
I'm sure it is common way. From Wikipedia:
> A mobile country code (MCC) is used in combination with a mobile network code (MNC) (a combination known as an "MCC/MNC tuple") to uniquely identify a mobile network operator (carrier) using the GSM (including GSM-R), UMTS, and LTE public land mobile networks.